### PR TITLE
Update Grafana to 4.6.3

### PR DESCRIPTION
**Pull request by jeffmendoza**
Switch from using custom Docker image to the upstream image and a configmap with sidecar. Config files are now under install/kubernetes/templates/addons/grafana-config and are inserted by install/updateVersion.sh.

### CODE REVIEW

File: install/kubernetes/addons/grafana.yaml [hunk 1]
```diff
@@ -28,25 +28,1513 @@ spec:
     spec:
       containers:
       - name: grafana
-        image: gcr.io/istio-testing/grafana:5253b6b574a98b209c0ef3d0d6e90c1b8d6a5c2a
+        image: grafana/grafana:4.6.3
```
Comment (douglas-reid) [inline install/kubernetes/addons/grafana.yaml]: :+1:

File: install/kubernetes/templates/addons/grafana-config/custom.ini [hunk 2]
```diff
@@ -0,0 +1,9 @@
+[auth]
+disable_login_form = true
```
Comment (douglas-reid) [inline install/kubernetes/templates/addons/grafana-config/custom.ini]: aside: we should probably revisit this at some point, especially as these start to get exposed via ingress.

Comment (jeffmendoza) [reply install/kubernetes/templates/addons/grafana-config/custom.ini]: Agreed. Until then, I like having this as a configmap as it is easier for users to update/customize from a local file/directory.

**Comment by jeffmendoza** [comment]
Rebased to see if tests pass now. Might need another ok-to-test

**Pull request merged**